docs/source/backends/adios2.rst

```diff
@@ -105,6 +105,14 @@ The new layout may be activated **for experimental purposes** in two ways:
 
 The ADIOS2 backend will automatically recognize the layout that has been used by a writer when reading a dataset.
 
+.. tip::
+   This schema does not use ADIOS2 attributes any more, making the ``bpls -a`` and ``bpls -A`` switches useless for openPMD datasets.
```

Comment:
I think this needs an empty newline to render properly.

Maybe also a suggestion in the meaning:
```suggestion

   This schema does not use ADIOS2 attributes anymore, thus ``bpls -a`` and ``bpls -A`` attribute switches do not show openPMD attributes.
```